.chloggen/bc--optimizesnappybufferallocation.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I would say that this is not a bug because it is not affecting the functionality. This is an enhancement.